pom.xml

```diff
@@ -17,12 +17,12 @@
     <inceptionYear>2005</inceptionYear>
     <url>https://github.com/mojohaus/nbm-shared</url>
     <issueManagement>
-        <system>jira</system>
-        <url>http://jira.codehaus.org/browse/MNBMODULE</url>
+        <system>GitHub</system>
+        <url>https://github.com/mojohaus/nbm-shared/issues</url>
     </issueManagement>
-    <prerequisites>
+    <!--<prerequisites>
```

Comment:
comment out or remove?

Reply:
this should be removed because nbm-shared is not a maven plugin